WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -488,7 +488,11 @@ local methods = {
             end
           end
         else
-          WeakAuras.PickDisplay(data.id);
+          if (WeakAuras.IsDisplayPicked(data.id)) then
+            WeakAuras.ClearPicks(data.id);
```

Comment:
So this code is almost the same as the above one for IsCtrlKeyDown(). It might be easier to combine them.

And also I'm not sure how the interactions between normal clicks and ctrl+clicks should be, but I think it should be:

 _ | Normal Click | Ctrl Click
------------ | ------------- |  -------------
Unselected  |  Clear+Select | Add to selection
Single-Selection | Unselect (1) | Unselect
Multi-Selection | Clear + Select (2) | Unselect

Clear+Select = The clicked one is the only selected after the click.

(1) Is the one that is a bit off, because looking at explorer, that's now how it usually works. But we need that, so I'm okay with it.

(2) Is how explorer works, and different from what you have implemented.

But, I'm not sure how it behaved before, and you can convince me that what you have implemented is better or more consistent with how it worked before.